### ISSUE TITLE
Configure imaginary endpoint from Nextcloud's interface

### ISSUE THREAD

**Issue by devnoname120** [nextcloud/server#32614]
I use a [Hetzner Storage Share](https://www.hetzner.com/storage/storage-share) instance so I unfortunately don't have access to `config.php` and I can't install `imaginary` locally.

I have a Hetzner VPS as well though, on which I would like to run imaginary in addition to Collabora.

Could you add a way to manually configure an external `imaginary` endpoint from the Nextcloud interface, as well as making it possible to configure the supported file types from here?

Thanks a lot 🙏

P. S. — I can't just install Nextcloud on my VPS because local storage is 40 GB, and mounting my Hetzner Storage Share's Nextcloud as an external storage is too slow.

### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

**Is your feature request related to a problem? Please describe.**

I use a [Hetzner Storage Share](https://www.hetzner.com/storage/storage-share) instance so I unfortunately don't have access to `config.php` and I can't install `imaginary` locally.

**Describe the solution you'd like**

I have a Hetzner VPS as well though, on which I would like to run imaginary in addition to Collabora.

Could you add a way to manually configure an external `imaginary` endpoint from the Nextcloud interface, as well as making it possible to configure the supported file types from here?

**Describe alternatives you've considered**

I can't just install Nextcloud on my VPS because local storage is 40 GB, and mounting my Hetzner Storage Share's Nextcloud as an external storage is too slow.

**Additional context**

![image](https://user-images.githubusercontent.com/2824100/170564719-29e55ac6-072c-41eb-8e09-d4efcb29574b.png)

**Comment by CarlSchwan** [comment]
Good idea :) I'll see if I can find some time to implement this at some point.

**Comment by devnoname120** [comment]
> Good idea :) I'll see if I can find some time to implement this at some point.

Hetzner Storage Share is 3.90€/month for 1 TB RAIDed, 10 days of daily backups, 10 Gbit up/down.

This is a crazy low price! I can't find a reliable VPS with 1 TB at a reasonable price.

Thanks a lot 🙏

**Comment by rikmeijer** [comment]
How is this going, I am looking for this feature for the same reasons 🙂
